skyplane/utils/networking_tools.py

```diff
@@ -0,0 +1,37 @@
+import requests
+import re
+
+
+def get_ip() -> str:
+    """Get the IP address of the current machine."""
+    try:
+        ip = requests.get("https://api.ipify.org").text
+    except:
+        return None
+    return ip
+
+
+def get_cloud_region(ip: str, provider: str = "aws") -> str:
```

Comment:
We should ask users to pass this directly instead of inferring. It's OK to add another CLI argument.